lms/views/lti/deep_linking.py

```diff
@@ -123,7 +124,12 @@ def file_picker_to_form_fields_v13(self):
             "https://purl.imsglobal.org/spec/lti/claim/message_type": "LtiDeepLinkingResponse",
             "https://purl.imsglobal.org/spec/lti/claim/version": "1.3.0",
             "https://purl.imsglobal.org/spec/lti-dl/claim/content_items": [
-                {"type": "ltiResourceLink", "url": document_url}
+                {
+                    "type": "ltiResourceLink",
+                    "url": self.request.route_url("lti_launches"),
```

Comment:
D2L doesn't seem to like URL that are different in any way to the LTI launch one. That means no query parameters either.

A solution could be to send us back the URL as a custom parameter. This looks nicer than the URL approach, no more encoding issues.